app/addons/news/components.js

```diff
@@ -11,14 +11,78 @@
 // the License.
 
 import React from "react";
+import Cookies from 'universal-cookie';
```

Comment:
I'm not sure about adding a new dependancy to manage this? What about rather using localstorage to store this?

Reply:
that would also work, I suggested cookies because of how this is usually done in authentication / remember me type situations, but localStorage is fine, too